simulator-spring-boot/src/test/java/org/citrusframework/simulator/service/criteria/TestResultCriteriaTest.java

```diff
@@ -89,16 +89,16 @@ void testErrorMessage() {
     }
 
     @Test
-    void testFailureStack() {
-        assertNull(criteria.getFailureStack());
+    void testStackTrace() {
```

Comment:
Optional: 
I don't like getter/setter tests 😇

They make the source-code rigid.

Reply:
it's not necessarily about the getter/setter, but about the `stackTrace` method - it is (in contrary to the getters and setters) non-nullable.